swap_meet/item.py

```diff
@@ -0,0 +1,7 @@
+
+class Item:
```

Comment:
👍  I like your decision to make `condition` be a child attribute instead of a parent attribute. Either would have worked, but interesting to see a different decision!